pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1499,6 +1495,28 @@ func (n *NetworkPolicyController) syncAppliedToGroup(key string) error {
 	return nil
 }
 
+// getAppliedToWorkloads returns a list of workloads like Pods and ExternalEntities matching an AppliedToGroup
+// for standalone selectors or corresponding to a ClusterGroup.
+func (n *NetworkPolicyController) getAppliedToWorkloads(g *antreatypes.AppliedToGroup) ([]*v1.Pod, []*v1alpha2.ExternalEntity) {
+	// Check if an internal Group object exists corresponding to this AppliedToGroup.
+	intGroup, found, _ := n.internalGroupStore.Get(string(g.UID))
+	if found {
+		var pods []*v1.Pod
+		ig := intGroup.(*antreatypes.Group)
+		// Generate the list of Pods based on the GroupMembers set in the internal Group.
+		for _, gm := range ig.GroupMembers {
```

Comment:
similar to Quan's comment in the other PR, is racing a possibility here? i.e. An update in membership of a ClusterGroup is underway when this is evaluated

Reply:
yup .. i plan to address the other PR first and resolve all comments as there is overlap.